### ISSUE TITLE
Disconnections on flaky networks + no reconnect

### ISSUE THREAD

**Issue by Lenne231** [Temasys/AdapterJS#198]
We have problems with disconnections on a little bit flaky networks with the WebRTC plugin (0.8.870) but not when using Google Chrome (50.0.2661.102 m). 
Furthermore Google Chrome tries to reconnect if the ICE connection state has changed to "disconnected" but the WebRTC plugin does not.
Are there any differences in the implementation? I thought that the plugin and Google Chrome share the same codebase.

**Comment by johache** [comment]
Disconnections on bad networks are really dependent on luck, so there is not much for us to do.
The auto-reconnection however is relevant, we will look into it.

Thanks

**Comment by Lenne231** [comment]
Yes, but i was wondering how many disconnections we have on the same network. With the plugin we have about 2-3 disconnections in 10 minutes, with Google Chrome we have no disconnections in hours.

We are working on an iOS App using the native WebRTC API from Google and have the same problems there. So it doesn't seem to be related to the the plugin only.

Thanks
